src/AppInstallerCLICore/Commands/DebugCommand.cpp

```diff
@@ -148,6 +154,186 @@ namespace AppInstaller::CLI
                 "  </data>" << std::endl;
         }
     }
+
+    std::vector<Argument> ShowSixelCommand::GetArguments() const
+    {
+        return {
+            Argument{ "file", 'f', Args::Type::Manifest, Resource::String::SourceListUpdatedNever, ArgumentType::Positional },
+            Argument{ "aspect-ratio", 'a', Args::Type::AcceptPackageAgreements, Resource::String::SourceListUpdatedNever, ArgumentType::Standard },
+            Argument{ "transparent", 't', Args::Type::AcceptSourceAgreements, Resource::String::SourceListUpdatedNever, ArgumentType::Flag },
+            Argument{ "color-count", 'c', Args::Type::ConfigurationAcceptWarning, Resource::String::SourceListUpdatedNever, ArgumentType::Standard },
+            Argument{ "width", 'w', Args::Type::AdminSettingEnable, Resource::String::SourceListUpdatedNever, ArgumentType::Standard },
+            Argument{ "height", 'h', Args::Type::AllowReboot, Resource::String::SourceListUpdatedNever, ArgumentType::Standard },
+            Argument{ "stretch", 's', Args::Type::AllVersions, Resource::String::SourceListUpdatedNever, ArgumentType::Flag },
+            Argument{ "repeat", 'r', Args::Type::Name, Resource::String::SourceListUpdatedNever, ArgumentType::Flag },
+            Argument{ "out-file", 'o', Args::Type::BlockingPin, Resource::String::SourceListUpdatedNever, ArgumentType::Standard },

```

Comment:
Nit: Probably not too important for a debug command, but the arg types and strings just feel wrong...

Reply:
Yes, the goal was to not introduce any new arguments just for debugging purposes.  Rather than trying to find good type name fits, I just let autocomplete decide for me.  I then cheekily chose the string "Never" for the descriptions.

If the arbitrary type names are too much, I could define macros to make it easier to read.

Reply:
And that is what I did.